.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,60 @@
+name: Bug report
+description: You've found a bug on the documentation website
+labels: ['bug']
```

Comment:
Our old templates did not automatically add a label, that's fixed now with the new forms. 😄